docs/faq.md

```diff
@@ -98,11 +159,12 @@ We currently support adding a custom directory with an **incubating** feature. T
 
 ### Can I build to a local Docker daemon?
 
-See [`jib:dockerBuild` for Maven](../jib-maven-plugin#build-to-docker-daemon) and [`jibDockerBuild` for Gradle](../jib-gradle-plugin#build-to-docker-daemon).
-
-You can also [`docker pull`](https://docs.docker.com/engine/reference/commandline/pull/) the image built with Jib to have it available in your local Docker daemon.
+There are several ways of doing this:
 
-You can also [run a local Docker registry](https://docs.docker.com/registry/deploying/) and point Jib to push to the local registry.
+- Use [`jib:dockerBuild` for Maven](../jib-maven-plugin#build-to-docker-daemon) or [`jibDockerBuild` for Gradle](../jib-gradle-plugin#build-to-docker-daemon) to build directly to your local Docker daemon.
+- Use [`jib:buildTar` for Maven]() or [`jibBuildTar` for Gradle]() to build the image to a tarball, then use `docker load --input` to load the image into Docker (the tarball built with these commands will be located in `target/jib-image.tar` for Maven and `build/jib-image.tar` for Gradle by default).
+- [`docker pull`](https://docs.docker.com/engine/reference/commandline/pull/) the image built with Jib to have it available in your local Docker daemon.
+- Alternatively, instead of using a Docker daemon, you can [run a local Docker registry](https://docs.docker.com/registry/deploying/) and point Jib to push to the local registry.
```

Comment:
_run a local registry, such as a Docker registry, and point …_

(there are other implementations which may be better for their needs, like Nexus, Artifactory, etc.)